product_template_multi_link/tests/test_product_template_link.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplateLink(TransactionCase):
```

Comment:
You can inherit from `BaseCommon` class and remove the `tracking_disable=True` key from context set in `setUpClass` method